app/helpers/desk_helper.rb

```diff
@@ -0,0 +1,14 @@
+module DeskHelper
+
+  def desk_class(desk)
+    desk.occupied? ? 'danger' : 'success'
```

Comment:
I'm OK with this but in general I try to go for better readability than for less LOC. Also, I tend to leave ternary operators for a quick dynamic evaluation of an attribute and keep method implementations as readable as possible.

As I said, I'm OK with this. Just wanted to give my thoughts on the subject.